docs/use_cases/otel/grafana.md

```diff
@@ -0,0 +1,148 @@
+# Grafana Integration
+
+Next step is to integrate with Grafana so that we can query Feldera for these insights.
+
+We plan to create a Grafana dashboard like the one shown below:
+
+![Grafana Dashboard](grafana-dashboard.png)
+
+This dashboard takes inspiration from the workshop: [Analyzing and Visualizing OpenTelemetry Traces with SQL - John Pruitt, Timescale](https://www.youtube.com/watch?v=rjJgZ_MNmPQ)
+
+#### Step 1: Install the Feldera Grafana Plugin
+
+Since the Feldera datasource plugin is not yet available in the Grafana marketplace, it must be **manually installed** from [GitHub](https://github.com/feldera/grafana-datasource). You can download the latest release and unzip in in the plugins directory of your Grafana server.
+
+#### Step 2: Add Feldera as a Data Source in Grafana
+
+Steps to add a new data source:
+7. Open the Grafana Dashboard and go to **Connections -> Add New Connection**
+8. Search for **Feldera** and select the **Feldera Data Source Plugin**.
+9. Click **"Add a new data source"**.
+
+#### Step 3: Configure the Feldera Data Source
+
+![Feldera Data Source Config](feldera-data-source.png)
+
+##### Required Fields:
+
+| Field         | Value                                                            |
+| ------------- | ---------------------------------------------------------------- |
+| Base URL      | `http://feldera:8080` _(If running in Docker alongside Grafana)_ |
+| Pipeline Name | `otel` _(Your Feldera pipeline name)_                            |
+| API Key       | _(If authentication is required, provide the API Key)_           |
+
+Click **"Save & Test"** to verify the connection.
+
+## Creating a Grafana Dashboard
+
+Once Feldera is set up as a data source, you can build a **custom Grafana dashboard** to visualize OpenTelemetry data.
+
+For all the visualizations we create here, use the **feldera** data source.
+
+#### Requests Count
+
+This visualization displays the total number of requests in the selected time range.
+Visualization Type: **Stat**
+
+
+```sql
+-- requests count
```

Comment:
who runs this query?
this is parameterized SQL.
it seems that Grafana runs this.